### ISSUE TITLE
filter with contains for partial matches

### ISSUE THREAD

**Issue by nomoney4me** [neo4j-graphql/neo4j-graphql-js#149]
In graphiql, the follow works great when dealing with neo4j-graphql
```
{
  People(filter: {Aliases_contains: "Jesus"}) {
    Person_Lookup
    Name
    Aliases
  }
}
```
```
{
  "data": {
    "People": [
      {
        "Person_Lookup": "Jesus.904",
        "Name": "Jesus",
        "Aliases": "Jesus,Justus"
      },
      {
        "Person_Lookup": "Jesus.905",
        "Name": "Jesus",
        "Aliases": "Jesus Christ,Immanuel,Emmanuel,Christ,Jesus,God the Son,Word of God"
      },
      {
        "Person_Lookup": "Joshua.1727",
        "Name": "Joshua",
        "Aliases": "Joshua,Oshea,Jehoshua,Hoshea,Jeshua,Jesus"
      }
    ]
  }
}
```
However, when using `makeAugmentedSchema` with neo4j-graphql-js, I do not have the filter and results in "Unknown argument "filter" on field "People" of type "Query".

Is there any way to get a filter going? I've been reading the documentations but not much shows a solution for this.

The follow works great but I am hoping to get a partial match for Aliases rather than an exact match.
```
{
  People(Name: "Jesus") {
    Person_Lookup
    Name
    Aliases
  }
}
```
```
{
  "data": {
    "People": [
      {
        "Person_Lookup": "Jesus.904",
        "Name": "Jesus",
        "Aliases": "Jesus,Justus"
      },
      {
        "Person_Lookup": "Jesus.905",
        "Name": "Jesus",
        "Aliases": "Jesus Christ,Immanuel,Emmanuel,Christ,Jesus,God the Son,Word of God"
      }
    ]
  }
}
```

**Comment by johnymontana** [comment]
hey @nomoney4me - the newly released v2.5.0 of neo4j-graphql.js adds support for these filter arguments, similar to the Neo4j GraphQL database plugin. Could you give it a try and give us any feedback you have? See: https://grandstack.io/docs/neo4j-graphql-js.html#filtering

**Comment by johnymontana** [comment]
Closing as I hope the new filter functionality addresses your issue. Please reopen if the filter feature described above does not address your use case.
